shared/macros/10-bash.jinja

```diff
@@ -10,7 +10,10 @@ Then, assume that variables of that names are defined and contain the correct va
 #}}
 {{%- macro bash_instantiate_variables() -%}}
 {{%- for name in varargs -%}}
-{{{ name }}}='(bash-populate {{{ name }}})'
+{{{ name }}}=$(cat << EOF
+(bash-populate {{{ name }}})
+EOF
+)
```

Comment:
I would prefer instead of complicating the generated bash code we can improve the generator so that it would escape the quotes.

I think it could be added here:
https://github.com/ComplianceAsCode/content/blob/5f28a9ae08085e023c5af34876c518b45e85308f/ssg/build_remediations.py#L649

Reply:
This actually cannot work:
Basically we have this:
~~~
{{{ bash_instantiate_variables("login_banner_text") }}}
~~~

which makes the python code (after jinja substitution) generate this:
~~~
login_banner_text='<xccdf-1.2:sub idref="xccdf_org.ssgproject.content_value_login_banner_text" use="legacy"/>'
~~~
instead we would need this below:
~~~
login_banner_text='<xccdf-1.2:sub idref="xccdf_org.ssgproject.content_value_login_banner_text_escaped" use="legacy"/>'
~~~
but `content_value_login_banner_text_escaped` cannot exist, because it would require fixing the XML model to generate such var when user in SCAP workbench sets a value for `content_value_login_banner_text`.

Reply:
Actually the initial PR works fine, it's just that I made a mistake when creating a Test banner (I replaced spaces by `[\s\n]` instead of `[\s\n]+` ...

With this one below it works fine:
~~~
^L'accès[\s\n]+au[\s\n]+système[\s\n]+est[\s\n]+fourni[\s\n]+par[\s\n]+la[\s\n]+société[\s\n]+XXX[\s\n]+YYY[\s\n]+à[\s\n]+des[\s\n]+fins[\s\n]+d'activités[\s\n]+autorisées\.$
~~~